.github/workflows/action-ros-ci-with-setup-ros.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and test(actions-ros + setup-ros on ubuntu:focal)
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ubuntu:focal
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: setup ROS environment
+        uses: ros-tooling/setup-ros@v0.1
+        with:
+          required-ros-distributions: foxy
+
+      - name: action-ros-ci
+        uses: ros-tooling/action-ros-ci@v0.1
+        with:
+          package-name: obstacle_stop_planner_refine
+          target-ros2-distro: foxy
+          vcs-repo-file-url: build_depends.repos
```

Comment:
[Question] Could you explain what commands are executed with these settings for my understanding? (for example by referring to the action's code)

Reply:
actions-ros performs `colcon build + colcon test + colcon lcov-result + colcon coveragepy-result`
You can check commands here: <https://github.com/tier4/obstacle_stop_planner_refine/runs/2029944928?check_suite_focus=true>

![image](https://user-images.githubusercontent.com/19993104/110048176-ad3bb280-7d92-11eb-95b8-97ebbf6273ae.png)